taskvine/src/bindings/python3/ndcctools/taskvine/dask_dag.py

```diff
@@ -56,7 +57,7 @@ def containerp(s):
     def symbolp(s):
         return isinstance(s, dts.DataNode)
 
-    def __init__(self, dsk):
+    def __init__(self, dsk, reconstruct=False, merge_size=2):
```

Comment:
Please add detailed comments of what these options do.

Reply:
Added comments on how the graph is reconstructed.